### DISCUSSION TITLE
Region mapping in iXXXXX format

### DISCUSSION THREAD

**Discussion by YW-nee** [NREL/ReEDS-2.0#128] [Q&A: Data] [answered]
I'm looking at the capacity file: wind-ons_exog_cap_site_reference.csv for MW builds, the region column uses iXXXXX format. is there a mapping file that maps them to P and S formats of regions ? thank you.

**Comment by patrickbrown4** [comment, marked as answer]
Yes, the inputs/rsmap.csv file maps from individual wind sites (iXXXXX for onshore and oXXXXXX for offshore) to the 134 p regions:
https://github.com/NREL/ReEDS_OpenAccess/blob/main/inputs/rsmap.csv

**Reply by YW-nee** [reply]
Thank you very much @patrickbrown4 !
